packages/sdk/src/types/runtime/user.ts

```diff
@@ -16,6 +16,11 @@ export interface UserLike {
     name: string;
     transform: TransformLike;
     subscriptions: SubscriptionType[];
+
+    /**
+     * A grab bag of miscellaneous, possibly host-dependent, properties.
+     */
+    properties: { [name: string]: any };

```

Comment:
Can we document the `remoteAddress` property here? What would that syntax look like?